lib/src/components/NxToast/__tests__/NxToast.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { fireEvent, screen } from '@testing-library/react';
+
+import userEvent from '@testing-library/user-event';
+
+import NxToast from '../NxToast';
+import { NxToastProps } from '../types';
+import { NxErrorAlert } from '../../NxAlert/NxAlert';
+
+describe('NxToast', function() {
+  const simpleProps: NxToastProps = {
+    onClose: () => {},
+    children: <NxErrorAlert>Toast Message</NxErrorAlert>
+  };
+
+  const quickRender = rtlRender(NxToast, simpleProps),
+      renderEl = rtlRenderElement(NxToast, simpleProps);
+
+  it('renders a div with a class .nx-toast', function() {
+    const component = quickRender();
+    const toast = component.container.children[0];
```

Comment:
The purpose of `rtlRenderElement` (and thus your `renderEl`) is to simplify this
```suggestion
    const toast = renderEl()!;
```